ngsi_adapter/src/.gjslintrc

```diff
@@ -5,5 +5,5 @@
 --nomultiprocess
 --debug_indentation
 --time
---disable=110,212,217
---custom_jsdoc_tags=module,function,callback,abstract,constant,augments,memberof,returns
+--disable=1,110,212,217
```

Comment:
Space between `function` keyword and parentheses, in anonymous functions. Deliberately this should **not** be a lint error.